http.lua

```diff
@@ -606,11 +606,23 @@ function M.send(method, t)
             addr = host
             port = 443
         end
+    elseif schema == "unix" then
+        connect = socket.connect
+        schema, uds, http_path = t.url:match("^(.*)://(.-):(.*)$")
+        host = "docker"
```

Comment:
Not a maintainer, but this does not look correct to me.

Reply:
Yeah, this doesn't look generic enough. Also, ports are not used by UNIX domain sockets.

However, since the underlying socket class implementation does support UDS, and you have use case for it, we can add it. Would you care to clean up this patch a bit or maybe we should implement it (in that case we'd like to have some feedback from you)

Reply:
Yeah, this doesn't look generic enough. Also, ports are not used by UNIX domain sockets.

However, since the underlying socket class implementation does support UDS, and you have use case for it, we can add it. Would you care to clean up this patch a bit or maybe we should implement it (in that case we'd like to have some feedback from you)